docs/README.md

```diff
@@ -9,4 +9,5 @@ The nf-core/mhcquant documentation is split into the following files:
     * [bwForCluster BinAC](configuration/binac.md)
```

Comment:
Remove this line because you have deleted `binac.md`

Reply:
This file should look like:
https://github.com/nf-core/tools/edit/master/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/docs/README.md

Reply:
I tried to adapt the docs/README a bit since for example I don't make use of a reference genome. But if you require it I can replace it with the standard.

Reply:
I thought that might rather be confusing for users of my pipeline